src/_posts/platform/app/2000-01-01-tailscale.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: Configure Tailscale
+modified_at: 2025-12-31 00:00:00
+tags: tailscale networking vpn buildpack
+---
+
+[Tailscale](https://tailscale.com/) is a zero-config VPN that creates a secure network between your servers, computers, and cloud instances. This guide explains how to connect your application container on Scalingo to your Tailscale network.
+
+## Prerequisites
+
+* A Tailscale account.
+* A [Tailscale Auth Key](https://tailscale.com/kb/1085/auth-keys/).
+
+## Step 1: Multi-buildpack Configuration
+
+To install Tailscale and your application dependencies, you need to use the [multi-buildpack]({% post_url platform/deployment/buildpacks/2000-01-01-multi %}).
+
+Create or update the `.buildpacks` file at the root of your repository:
+
+```text
+https://github.com/Scalingo/apt-buildpack.git
+# Add any other buildpack required to build your application (e.g. scalingo/nodejs, scalingo/ruby, etc.)
+```
+
+## Step 2: Aptfile Configuration
+
+The [APT buildpack]({% post_url platform/deployment/buildpacks/2000-01-01-apt %}) allows you to install additional packages. Create a file named `Aptfile` at the root of your repository with the following content:
+
+```text
+wget
+curl
+tailscale
+:repo:deb [trusted=yes] https://pkgs.tailscale.com/stable/ubuntu jammy main
+```
+
+## Step 3: .profile Configuration
+
+The `.profile` script is executed before your application starts. It will be used to start the Tailscale daemon and connect to your network.
```

Comment:
suggestion: I would add a link to the dedicated section about this behaviour (https://doc.scalingo.com/platform/internals/container-management#the-profile-script).